src/test/java/PlayListTests.java

```diff
@@ -42,13 +34,13 @@ public void deletePlaylist() {
         loginPage.login();
         Assert.assertTrue(homePage.isUserAvatarDisplayed());
         //WHEN
-        playListPage.deletingPlaylist(playListName);
+        playListPage.deletingPlaylist();
         //THEN
-        Assert.assertEquals(playListPage.getConfirmationMessage(),true);
+        Assert.assertEquals(playListPage.getConfirmationMessage(), true);
     }
 
     @Test(enabled = true, priority = 2)
-    public void createNameTwice () {
+    public void createNameTwice() {
```

Comment:
Test failed!

You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/211770191-c50b5687-486a-4729-a7b8-bc16f0804511.png)

Reply:
I also corrected this one too.